app/livechat/imports/server/rest/visitors.js

```diff
@@ -47,18 +48,17 @@ API.v1.addRoute('livechat/visitors.chatHistory/room/:roomId/visitor/:visitorId',
 		});
 		const { offset, count } = this.getPaginationItems();
 		const { sort } = this.parseJsonQuery();
-
 		const history = Promise.await(findChatHistory({
 			userId: this.userId,
 			roomId: this.urlParams.roomId,
 			visitorId: this.urlParams.visitorId,
+			text : this.queryParams.searchText,
```

Comment:
Let's add a new optional parameter(closedChatsOnly - boolean) here to fetch only closed conversations, okay?

Reply:
added